pkg/store/errors.go

```diff
@@ -16,12 +16,12 @@ limitations under the License.
 
 package store
 
-import "errors"
+import "github.com/containerd/containerd/errdefs"
 
 var (
 	// ErrAlreadyExist is the error returned when data added in the store
 	// already exists.
-	ErrAlreadyExist = errors.New("already exists")
+	ErrAlreadyExist = errdefs.ErrAlreadyExists
 	// ErrNotExist is the error returned when data is not in the store.
-	ErrNotExist = errors.New("does not exist")
+	ErrNotExist = errdefs.ErrNotFound
```

Comment:
Thinking out loud: wondering if the abstraction should be kept, or if these should be marked deprecated and locations that use these errors to use the `containerd/errdefs` package directly

Reply:
I'm ok removing the abstraction. I was hoping @Random-Liu would chime in on an approach as well. So I did this as the least invasive to the overall project references.

Reply:
Definitely not urgent, and I agree on keeping the diff for this PR small; 

if it's decided to deprecate (for a follow-up), these aliases can be kept (to not break potential consumers of this package), but marked "deprecated" (motivating them to switch to the containerd/errdefs package).

Reply:
Yep I can do that as a followup to this PR

Reply:
SGTM to just mark them as deprecated in the comments along with a TODO: to remove in 1.5

Reply:
Sounds good. Are we ok with this PR or should we wait for @Random-Liu ?

Reply:
can you add the comment deprecating it with the TODO?

Reply:
My bad. I misunderstood this to be a followup PR. Done now.

Reply:
:-)